src/ORM/Connect/DBSchemaManager.php

```diff
@@ -528,7 +528,7 @@ public function requireIndex($table, $index, $spec)
             // Updated index
             $this->transAlterIndex($table, $index, $spec);
             $this->alterationMessage(
-                "Index $table.$index: changed to $specString <i class=\"build-info-before\">(from $oldSpecString)</i>",
+                "Index $table.$index: changed to '$specString' <i class=\"build-info-before\">(from '$oldSpecString')</i>",
```

Comment:
Unrelated but it makes it easier to visually parse what's going on when indexes change
This change helped me debug